libsql/src/parser.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code)]
```

Comment:
When does this warning trigger? We could use `cfg_attr` to only enable this in certain feature flag situations.

Reply:
It's mostly happening since I moved the parser.rs as an entire file, however in context of hrana we're only using a small subset of its functions.